Parsers/ASimAuditEvent/Parsers/ASimAuditEventInfobloxBloxOne.yaml

```diff
@@ -0,0 +1,123 @@
+Parser:
+  Title: AuditEvent ASIM parser for Infoblox BloxOne
+  Version: '0.1.0'
+  LastUpdated: Jun 26 2024
+Product:
+  Name: Infoblox BloxOne
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM AuditEvent Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Infoblox BloxOne Documentation
+- Link: https://docs.infoblox.com/space/BloxOneThreatDefense/35406922/DNS+Query%2FResponse+Log+Message+Mapping
+Description: |
+  This ASIM parser supports normalizing AuditEvent logs from Infoblox BloxOne to the ASIM AuditEvent normalized schema. These events are captured through the Azure Monitor Agent (AMA) which are sent by the Data Connector Service of Infoblox BloxOne.
+ParserName: ASimAuditEventInfobloxBloxOne
+EquivalentBuiltInParser: _ASim_AuditEvent_InfobloxBloxOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: 
+  let EventSeverityLookup = datatable (LogSeverity:string, EventSeverity:string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let parser = (disabled:bool=false) {
+      CommonSecurityLog
+      | where not(disabled)
+          and DeviceVendor == "Infoblox"
+          and DeviceEventClassID has "AUDIT"
```

Comment:
DeviceEventClassID could be EventOriginalType?

Reply:
Done